core/src/main/java/org/apache/iceberg/GenericDataFile.java

```diff
@@ -165,7 +165,7 @@ public GenericDataFile(org.apache.avro.Schema avroSchema) {
    *
    * @param toCopy a generic data file to copy.
    */
-  private GenericDataFile(GenericDataFile toCopy) {
+  private GenericDataFile(GenericDataFile toCopy, boolean fullCopy) {
```

Comment:
Nit: Javadoc @params are now inconsistent.  I'd be fine with either removing the exiting comment or adding the new one.

Reply:
Fixed.